From 06cb528545e936d2eaad96080810b162f6978539 Mon Sep 17 00:00:00 2001
From: Christoffer Holmstedt <christoffer@christofferholmstedt.se>
Date: Fri, 21 Aug 2015 11:42:00 +0200
Subject: [PATCH] arm: dts: Update RPi B Plus dts for Openlabs 802.15.4
 transceiver

This adds the device tree settings for the Openlabs.co 802.15.4 SPI board to be
used with the RPi Model B plus board.

Signed-off-by: Christoffer Holmstedt <christoffer@christofferholmstedt.se>
---
 arch/arm/boot/dts/bcm2835-rpi-b-plus.dts | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/arch/arm/boot/dts/bcm2835-rpi-b-plus.dts b/arch/arm/boot/dts/bcm2835-rpi-b-plus.dts
index e479515099c3..696707e8bafc 100644
--- a/arch/arm/boot/dts/bcm2835-rpi-b-plus.dts
+++ b/arch/arm/boot/dts/bcm2835-rpi-b-plus.dts
@@ -28,3 +28,17 @@
 		brcm,function = <4>; /* alt0 */
 	};
 };
+
+&spi {
+	status = "okay";
+	at86rf233@0 {
+		compatible = "atmel,at86rf233";
+		spi-max-frequency = <6000000>;
+		reg = <0>;
+		interrupts = <23 1>;
+		interrupt-parent = <&gpio>;
+		reset-gpio = <&gpio 24 1>;
+		sleep-gpio = <&gpio 25 1>;
+		xtal-trim = /bits/ 8 <0x0F>;
+	};
+};
-- 
2.1.4

